### PR TITLE
Support additional default test options

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

**Review by portante — COMMENTED** [review]
Ready for review.

**Review by webbnh — APPROVED** [review]
👍

File: server/test/run [hunk 1]
```diff
@@ -294,20 +294,23 @@ function _setup_state {
...
+        # The individual test configs reference the global test config as
+        # `state-pbench-server.cfg`.  Copy the global test config into place,
+        # and then add the particular test config to final pbench-server.cfg.
+        cp ${_tdir}/state/config/pbench-server.cfg ${_testopt}/lib/config/state-pbench-server.cfg
+        cat ${_state_config} >> ${_testtmp}/pbench-server.cfg
+    else
+        # Since there is no particular test configuration, add the global test
+        # configuration to the final pbench-server.cfg without the duplicate
+        # `[DEFAULT]` header.
+        sed 1d "${_tdir}/state/config/pbench-server.cfg" >> ${_testtmp}/pbench-server.cfg
```
[the remark this answers is not in the document]
Comment (portante) [reply server/test/run:312]: We can fix that another time.

**Review by ndokos — COMMENTED** [review]
LGTM

**Pull request merged**